src/app/game-state/activity.service.ts

```diff
@@ -551,8 +577,13 @@ export class ActivityService {
         } else {
           this.logService.addLogMessage("You fumble with the wrong tools and materials and hurt yourself.","INJURY","EVENT");
           this.characterService.characterState.status.health.value -= this.characterService.characterState.status.health.max * 0.05;
+          this.mainLoopService.pause = true;
         }
       }],
+      resourceUse: [{
+        stamina: 100,
+        health: this.characterService.characterState.status.health.max * 0.05 + 1
```

Comment:
I don't think this will do what I think you want it to. I think it will set it to 5% of the health value when the activity object is created, not the current health value when it gets checked.